cmd/deploy.go

```diff
@@ -152,11 +161,12 @@ func InitRunCommand() (*cobra.Command, error) {
 			concurrencyDefault = 0
 		}
 	}
-	var concurrency int
-	run.Flags().IntVarP(&concurrency, "concurrency", "c", concurrencyDefault, "concurrency")
+	run.Flags().IntP("concurrency", "c", concurrencyDefault, "concurrency")
 
-	var clusterUrl string
-	run.Flags().StringVar(&clusterUrl, "url", os.Getenv("KUBERNETES_CLUSTER_URL"),
+	run.Flags().String("url", os.Getenv("KUBERNETES_CLUSTER_URL"),
 		"URL of the Kubernetes cluster. Overrides KUBERNETES_CLUSTER_URL env variable in AppController pod.")
+
+	run.Flags().StringP("replicas", "n", "1", "number of flow replicas: [+]number")
```

Comment:
> Don't you think that change of replica count and deletion of replicas should be separate command?
"kubeac run -n2" will make replica count be 2. If there was 1, 1 will be created. How do you separate it into two commands?

> Also, what does refresh do? What is the point of it?
I meant that no new replicas are going to be created or deleted. The outcome will be the same as if you do "kubeac run" before flows - it will just ensure that resources do exist and if they not for some reason - recreacte them again. 
This also close to default behavior of this parameter. If you don't specify -n parameter at all (or specify empty string) it will make replica count = +0 and minReplicaCount = 1, which means if there are no replicas it will create one, otherwise just refresh the resources. This is exactly what "kubeac run" does in master. So the backward compatibility retained